src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -712,16 +712,16 @@ void jit_gelu_tanh_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, con
 
     tanh_emitter->emit_code(
             { vmm_aux0.getIdx() },
-            out_vec_idxs,
+            { vmm_aux0.getIdx() },
```

Comment:
Just for my understanding: why cannot we directly use dst register here?

Reply:
We will lose value in `vmm_src` register (which is used below) if `vmm_dst` and `vmm_src` are the same register.